lib/alerts.py

```diff
@@ -13,8 +13,8 @@ def current_public(self):
         return [alert for alert in self if alert.is_current_public]
 
     @property
-    def expired_or_test(self):
-        return [alert for alert in self if alert.is_expired_or_test]
+    def expired(self):
```

Comment:
So if an operator channel message goes out, it will prompt people to go look at gov.uk/alerts. And when they get there, they will see nothing in current alerts and nothing in past alerts. At that point, what should they think? That the message they received is genuine?

I guess I just want to check that is definitely the decision we want?

Reply:
I agreed this with @quis, but now I'm not so sure. At the moment it makes sense because we manually change the site to say there are "planned tests". In future we don't want to be doing with that, so you're right there won't be any kind of confirmation about the alert until it expires. On the other hand, we don't want to alert people by saying "1 current alert" all in red. A complication is that these alerts don't have their own pages at the moment.

How about this (to avoid blocking this PR):

- We keep the change as-is (I'd argue it's a net improvement over treating all planned tests as "past alerts").

- We have a separate card to explore how to show planned tests when they happen. I'm thinking "1 current test", and letting the current-alerts page show them, potentially with more details e.g. the test area.

Reply:
I'm happy to go with your suggestion if @quis is

Reply:
Yup, this is good for now. We can still pre-announce planned tests manually for the moment, but at least they will go onto the past alerts page automatically with this strategy.

Reply:
OK, I'll merge this in.

Just to confirm, is it worth me writing a card to revisit this (as per my second bullet point)?

Reply:
Not yet. I don’t think we know enough about the frequency/nature of future testing.